src/renderer/components/Onboarding/Screens/SelectDevice/DeviceSelectorOption.js

```diff
@@ -7,8 +7,8 @@ import Text from "~/renderer/components/Text";
 
 const DeviceIllustrationContainer = styled.div`
   position: absolute;
-  bottom: 0;
-  right: 0;
+  top: 93px;
```

Comment:
Why?

Reply:
Cause export are only the images and not the one with padding as before.
@juan-cortes Do you want I ask another export of images or is it ok like this ?

Reply:
As long as there's a reason and it doesn't break other things, I dont mind.